catalog/app/components/SearchResults/SearchResults.js

```diff
@@ -492,6 +500,23 @@ function ObjectHit({ showBucket, hit: { path, versions, bucket } }) {
   )
 }
 
+function DirHit({
+  showBucket,
+  hit: {
+    path,
+    versions: [v],
+    bucket,
+  },
+}) {
+  const handle = { bucket, key: path }
+  return (
+    <Section>
+      <ObjectHeader {...{ handle, showBucket }} />
+      <Meta meta={v.meta} />
```

Comment:
I don't see meta for "dir" search query inside quilt-dev-null. How can I find directories with meta?

If there is no meta, ObjectHeader's margin-bottom appends to Section's padding-bottom, and space above and under dir's title are not equal.

Reply:
> How can I find directories with meta?

i'm afraid the only option is creating them)

> If there is no meta, ObjectHeader's margin-bottom appends to Section's padding-bottom, and space above and under dir's title are not equal.

yeah i see, i'll fix that, thx

Reply:
fixed